Update and rename 2023-12-20-FAQ.md to 2024-01-30-FAQ.md (#55)
get10101/10101.github.io: blog/2024-01-30-FAQ.md

### How is the DLC channel funded?
DLC channels are bilaterally funded.

We, on our side, will always put the necessary margin to cover the quantity (number of contracts) of your trade with a x2 leverage.

Examples:
- If the **opening price is $40.000** and you open a **$2.000 position** with a **x4 leverage**, your margin is **0,0125BTC**, ours is **0,025BTC**. 
**Total channel value : 0,037BTC**
- If the **opening price is $40.000** and you open a **$2.000 position** with a **x1 leverage**, your margin is **0,05BTC**, ours is **0,025BTC**. 
**Total channel value : 0,075BTC**
If the **opening price is $40.000** and you open **a $2.000 position** with a **x2 leverage**, the channel will be perfectly balanced as both parties will have a **0,025BTC margin**, for a **total channel value of 0,05BTC**

Conclusion : your margin in BTC will depend on the opening price, quantity, and on the leverage you choose. Ours will always remain relative to the opening price and quantity you have selected

> **As your first trade defines the configuration of your channel, it's important to take the above elements into account when assessing your payout expectations.**

Reviewer: > I mean, if that guy opened a new trade, we funded the other side of the channel with the corresponding margin, right?

Yeah, at the moment we are going with 50% of what they put up as margin. But that should be configurable soon.

> If that is correct : isn't our side of the channel the maximum payout he can expect as long as the channel is open as is? (if no splicing, no liquidity lending).

I now see where you are going with this! I wouldn't say it's the maximum payout, because that also includes their own margin i.e. payout is `margin +/- pnl - fees`. You could say that our side of the channel (minus order-matching fees) is the maximum amount they can earn with their first trade. And that amount changes as order-matching fees are paid to the coordinator.

Overall I think you are not wrong with what you wrote above and your comment, but maybe we can be a bit more explicit.

 

If this explanation isn't clear enough for you, please let us know!

We'll be publishing a detailed article on the financing of DLC channels and trading orders shortly.

### How can I open a DLC Channel?

By creating your first order, you will automatically create your DLC Channel with 10101.
You can close your position, but your channel will remain until you decide to close it.

Depending on the size of this channel, you'll be able to open larger or smaller positions in the trading section, or hold more or less synthetic currency in your wallet (to be released)

### Can I open several positions at the same time?

It is not yet possible to open several different positions.
However, it is in our plans to make it possible to resize your position at any time, without being able to change the leverage.

> If you want to resize it outside of the limits allowed by your DLC Channel, it will require a DLC channel resizing

Reviewer: ```suggestion
If you want to resize your position beyond the limits of your DLC Channel, you will have to resize your _channel_ (see next section).
```

### Can I resize my DLC Channel (splicing)?
It is not yet possible to open several different positions.

> However, it is in our plans to make it possible to resize your channel at any time for a fee (it will require on-chain operations), making it more cost-effective than closing and reopening a position.

Reviewer: Copy-paste error I suppose.

We can say: "No, but definitely in the future".

### Why can I not trade with my full off-chain balance?

You will not be able to open positions covering your entire off-chain balance, as you will need to keep a reserve.

Part of this reserve must remain so that on-chain transaction fees can be covered, enabling you to recover your funds in any worst-case scenario.

### Can I fund my 10101 wallet using Lightning?

> It is no longer possible, with our new version, to fund your wallet nor your DLC Channel using an Lightning.

Reviewer: ```suggestion
From version 1.8.0, it's no longer possible to fund your wallet nor your DLC Channel via Lightning.
```

> However, we intend to make this possible in a different way to what has been done before. Stay tuned!

Reviewer: ```suggestion
However, we are working on reintroducing this feature. Stay tuned!
```
